frontend/src/components/dashboard/demo/create-demo-modal/CreateDemoModal.module.scss

```diff
@@ -42,14 +35,21 @@ $modal-width: $tablet;
       justify-content: space-between;
       height: 4.5rem;
       border: 1px solid $primary;
+      cursor: default;
+
+      &.active:hover {
+        background: $primary;
+      }
 
       &:not(.active) {
         color: $primary;
-        background: $neutral-light;
+        background: transparent;
 
         &:hover {
-          background: $primary;
-          color: white;
+          background: rgba(181, 196, 255, 0.5);
```

Comment:
Could we rewrite this as `rgba($blue-100, 0.5)`so that we can infer the color from our variables

Reply:
I don't think it's possible to write it that way. Perhaps we should define a `$blue-100-translucent` variable in `src/styles/base/_colours.scss` instead?

Reply:
Ahh, disregard what I said. I didn't know that that was a SASS feature.